bin/server.js

```diff
@@ -17,14 +19,9 @@ function start () {
   })
 
   http.createServer((req, res) => mount(req, res)).listen(8080, () => {
```

Comment:
thank you, all looks good, just maybe change this fixed port number as well

Reply:
Missed that! Thanks.